inventory.adoc

```diff
@@ -4,7 +4,47 @@
 Production:: https://inventory.izettle.com
 
 ### Description
-The Inventory Service keeps track of the stock level of products. In the API clients can list current stock levels and update balance for products. The service automatically decrease the stock when a purchase is created. Currently the API is only available in the production environment requiring production credentials to access.
+The Inventory Service keeps track of the stock level for variants within products. The API clients can list current stock levels and update balance for products. The service automatically decrease the stock when a purchase is created. Currently the API is only available in the production environment requiring production credentials to access.
+
+For inventory we use a concept of transactions, where we always move stock between so called `Locations`.
+Locations are created automatically and at the moment, the locations that exists and the natural flow of stock:
+```
+                              >BIN
+SUPPLIER > STORE  > SOLD
+```
+`SUPPLIER` is an abstraction of a supplier and holds infinite stock.
+
+`STORE` is the location of the current stock for a variant.
+
+`BIN & SOLD` are abstractions for when a product is sold or thrown away.
+
+An important note is that you cannot directly set a stock balance for a variant.
+If you want to set an absolute value to a variant you need to calculate the change needed to reach that stock value.
+
+#### Example 1:
+`PUT /organizations/{organizationUuid}/inventory`
+
+ I have 30 stock in `STORE` and I want to set that to 20.
+ Then I move 10 stock to `SUPPLIER, SOLD or BIN` location (depend on your implementation), note that we always move positive numbers and.
+ ```json
+  "productUuid": <uuid>
+  "variantUuid": <uuid>,
+  "fromLocationUuid": <STORE_location_uuid>,
+  "toLocationUuid": “<SUPPLIER_location_uuid>,
+  "change": 10
+ ```
+#### Example 1:
+`PUT /organizations/{organizationUuid}/inventory`
+
+I have 30 in stock and want to set to 40, then we just swap the locations.
+ ```json
+  "productUuid": <uuid>
+  "variantUuid": <uuid>,
+  "fromLocationUuid": <SUPPLIER_location_uuid>,
+  "toLocationUuid": “<STORE_location_uuid>,
+  "change": 10
```

Comment:
Wouldn't a better example be supplier->store, or store->bin? The scenario above (hopefully) doesn't happen very often and might be misleading?

Reply:
Never mind. I see that you already changed (or I misread before). :D

Reply:
You probably read the first example, where i sent store -> supplier, which is a bad example, fixed that :) thanks!